modules/core/src/lapack.cpp

```diff
@@ -52,7 +52,7 @@ namespace cv
 
 int LU(float* A, size_t astep, int m, float* b, size_t bstep, int n)
 {
-    return hal::LU32f(A, astep, m, b, bstep, n);
+    return hal::LU32f(A, astep, m, b, bstep, n) ? true : false;
```

Comment:
the function is declared as int, however it returns bool; can you please fix it?